source/_patterns/03-templates/content-grid/content-grid.twig

```diff
@@ -0,0 +1,69 @@
+{% set gridBaseName = 'grid-content' %}
+{% set gridVariantName = '' %}
+{% if secondaryContent is defined %}
+  {% set gridVariantName = gridBaseName ~ '--has-secondary-content' %}
+{% else %}
+    {% set gridVariantName = gridBaseName ~ '--no-secondary-content' %}
+{% endif %}
+
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge([gridBaseName, gridVariantName]) }) -%}
+
+
+{% if isMain is defined and isMain %}
+  {% set element = 'main' %}
+  {% set attributes = attributes|merge({role: 'main' }) %}
+{% else %}
+  {% set element = 'div' %}
+{% endif %}
+
+<{{ element }} {%- include 'atoms-attributes' %}>
+
+  {%- with {content: tertiaryContent|default([])} only -%}
+    {%- block tertiaryContent -%}
+
+      {%- if content|length -%}
+
+        {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['grid-tertiary-content']) }) -%}
+
+        <div {%- include 'atoms-attributes' %}>
+          {%- include 'atoms-text' with {nodes: content} -%}
+        </div>
+
+      {%- endif -%}
+
+    {%- endblock tertiaryContent -%}
+  {%- endwith -%}
+
+  {%- with {content: primaryContent|default([])} only -%}
```

Comment:
Don't think `primaryContent` should be optional (removing the default reveals some of the uses have problems).